File: app/src/main/java/com/u1fukui/bbs/model/Notification.java
package com.u1fukui.bbs.model;


import java.io.Serializable;
import java.util.Date;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class Notification implements Serializable {

    private static final long serialVersionUID = -6826333503595117337L;

    public long id;

    public String url;

    public String message;

    public Date createdAt;
}

File: app/src/main/java/com/u1fukui/bbs/model/Notification.kt
package com.u1fukui.bbs.model


import se.ansman.kotshi.JsonSerializable
import java.io.Serializable
import java.util.*

@JsonSerializable
data class Notification(
        val id: Long,
        val url: String,
        val message: String,
        val createdAt: Date
) : Serializable {

    companion object {

        private const val serialVersionUID = -6826333503595117337L
    }
}
